src/main/java/org/opentripplanner/raptor/api/view/ArrivalView.java

```diff
@@ -114,9 +116,8 @@ default AccessPathView accessPath() {
 
   /* Transit */
 
-  /** @return true if transit arrival, otherwise false. */
   default boolean arrivedByTransit() {
-    return false;
+    return this.arrivedBy(PathLegType.TRANSIT);
```

Comment:
Readability would be improved by generally using static imports everywhere for `PathLegType` enum references, here and below.

Reply:
94a12e49def32a0c209ad36467d39c69ea0b98bb